KubeArmor/feeder/policyMatcher.go

```diff
@@ -908,6 +908,14 @@ func (fd *Feeder) UpdateMatchedPolicy(log tp.Log) tp.Log {
 				return log
 			}
 		} else if log.Type == "MatchedPolicy" {
+			if log.PolicyEnabled == tp.KubeArmorPolicyAudited {
+				if log.Action == "Allow" {
+					log.Action = "Audit"
```

Comment:
let's keep the original action like the block action.
-> Audit (Allow)

Reply:
Sure.